src/Thoth.Json.Net/Decode.fs

```diff
@@ -236,6 +236,7 @@ module Decode =
         fun path token ->
             if token.Type = JTokenType.Date then
                 try
+                    // TODO: Is correct to always use en-US here? (and in the "else" branch)
                     System.DateTime.Parse(token.Value<string>(), new  System.Globalization.CultureInfo("en-US")) |> Ok
```

Comment:
I needed to add this culture info, otherwise it would fail on my system.
Day and month are not always in the same order depending on the culture. Like that, we know what we are expecting.